src/script/interpreter.cpp

```diff
@@ -1434,16 +1434,22 @@ bool GenericTransactionSignatureChecker<T>::CheckLockTime(const CScriptNum& nLoc
     if (nLockTime > (int64_t)txTo->nLockTime)
         return false;
 
-    // Finally the nLockTime feature can be disabled and thus
-    // CHECKLOCKTIMEVERIFY bypassed if every txin has been
-    // finalized by setting nSequence to maxint. The
+    // Finally the nLockTime feature can be disabled in IsFinalTx()
```

Comment:
I'm not a huge fan of having function names in the comments, because the code might change over time. eg. theoretically there could be another calling path that also disables, right? 

in this case I don't think its a big deal esp bc its consensus code that seems _quite unlikely_ to change

Reply:
FWIW the rationale for adding it was specifically to cross reference these the two locations in the code, as they are more easily understood together.

I can definitely see why this is not worth the risk of stale comments, but couldn't think of a simpler way.

Maybe just mentioning `CHECKLOCKTIMEVERIFY` and `SEQUENCE_FINAL` is a good compromise, there are few false positives, and those are even less likely to change.

Reply:
> FWIW the rationale for adding it was specifically to cross reference these the two locations in the code, as they are more easily understood together.

fair enough. makes sense in this case